src/backend/app/projects/project_crud.py

```diff
@@ -58,96 +52,62 @@ async def create_project_with_project_info(
         },
     )
 
-    if not new_project_id:
+    if not project_id:
         raise HTTPException(status_code=500, detail="Project could not be created")
     # Fetch the newly created project using the returned ID
     select_query = f"""
```

Comment:
This seems unnecessary after the insert above.

You could just add the fields you need to the RETURNING clause and save a database call 👍

Reply:
Thank you for the suggestion! I have updated the query to include only the necessary fields in the RETURNING clause, eliminating the extra database call. Appreciate the feedback! 👍